packages/contracts/src/dollar/interfaces/IStakingShare.sol

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: Apache-2.0
-pragma solidity ^0.8.3;
+pragma solidity ^0.8.16;
+pragma solidity ^0.8.16;
```

Comment:
I fixed this error already in another PR.
```suggestion
pragma solidity ^0.8.16;
```

Reply:
fixed again